forwards_paper.tex

```diff
@@ -196,14 +196,40 @@ \section*{Results}
 %%%%%%
 \subsection*{Simulation benchmarks}
 
-\plr{Jaime and Kevin to describe simulation methods here (separately).}
-
-Comparison of simulation with/without msprime, using \simupop{}
-or maybe just a simple haploid simulation with 1000 QTL and stabilizing selection on a trait (say).
-
-Maybe an estimate of how long \emph{just} the ARG recording and simplification takes,
-so that then we can say how fast the simulator would have to be to do $10^6$ whole chromosomes for $10^7$ generations
-in a day.
+To measure the performance gains from ancestry recording, we ran simulations implemented in C++ (see above) with and
+without ancestry tracking.  The simulations without ancestry tracking introduced neutral mutations at a scaled mutation
+rate equal to the scaled recombination rate (\textit{e.g.}, $\theta = 4N\mu = \rho$, where $\rho = 4Nr$, $N$ is the
+diploid population size, $\mu$ is the
+mutation rate to neutral mutations (per gamete, per generation) and $r$ is the recombination rate per diploid per
+generation.  Both $\mu$ and $r$ are the means of a Poisson process and we used the infinitely-many sites mutation model
+\citep{Kimura1969-uc}.
+
+We varied $N \in (1000, 10000, 50000)$, $\rho \in (10^3, 10^4, 10^5)$ and ran a single replicate of all $(N,\rho)$
+combinations.  Deleterious mutations were introduced at rate $\rho/100$ per generation, drawing scaled selection
+coefficients ($2Ns$)
+from a Gamma distribution with a mean of -5 and a shape parameter of 1.  This distribution of fitness effects results in
+thousands of weakly-deleterious mutations segregating in the population, many of which drift to intermediate
+frequencies.
```

Comment:
we should perhaps emphasize that (a) this means you can't do the same thing with a coalescent sim and (b) these are mutations that actually have to be tracked; so we only save time by not needing to track the neutral ones

Reply:
Will do.